src/tests/Common/wasm-test-runner/WasmTestRunner.proj

```diff
@@ -1,6 +1,6 @@
 <!-- This project requires an explicit SDK version number because it is used on Helix,
       and global.json is not available. -->
-<Project Sdk="Microsoft.Build.NoTargets/1.0.53" DefaultTargets="WasmBuildApp">
+<Project Sdk="Microsoft.Build.NoTargets/3.5.6" DefaultTargets="WasmBuildApp">
```

Comment:
We can drop the version here. The version is set in `global.json:14:    "Microsoft.Build.NoTargets": "3.5.0",`.
```suggestion
<Project Sdk="Microsoft.Build.NoTargets" DefaultTargets="WasmBuildApp">
```

Reply:
<!-- This project requires an explicit SDK version number because it is used on Helix,
      and global.json is not available. -->